contracts/event/MultipleResultsEvent.sol

```diff
@@ -26,17 +26,17 @@ contract MultipleResultsEvent is NRC223Receiver, Ownable {
         uint8 resultIndex;
         uint consensusThreshold;
         uint arbitrationEndTime;
-        ResultBalance[11] balances;
+        ResultBalance[4] balances;
     }
 
-    uint16 private constant VERSION = 2;
+    uint16 private constant VERSION = 3;
     uint8 private constant INVALID_RESULT_INDEX = 255;
 
     uint8 private _numOfResults;
     uint8 private _currentRound = 0;
     uint8 private _currentResultIndex = INVALID_RESULT_INDEX;
     string private _eventName;
-    bytes32[11] private _eventResults;
+    bytes32[4] private _eventResults;
```

Comment:
shall we create constants for `3` and `3+1`?

Reply:
cant use constants to declare array sizes